bin/pa11y.js

```diff
@@ -203,6 +241,14 @@ function requireFirst(stack, defaultReturn) {
 	}
 }
 
+/**
+ * Calculates if there are any errors to be reported (i.e. above the threshold)
+ * This is used for example to return the correct status code on the CLI
+ * @param {String} level - The level of reporting required (warning, notice, etc.)
+ * @param {Object} issues - The issues found by the test
```

Comment:
The parameter is named `results`
```suggestion
 * @param {Object} results - The results found by the test
```